---
title: Compiler Error CS0156
ms.date: 07/20/2015
f1_keywords:
- CS0156
helpviewer_keywords:
- CS0156
ms.assetid: 32026b1b-bcd7-4464-b63f-3b38c00452a6
ms.openlocfilehash: f310b4b2b8aa320146d9114a999a82dcae2bb528
ms.sourcegitcommit: 9b552addadfb57fab0b9e7852ed4f1f1b8a42f8e
ms.translationtype: MT
ms.contentlocale: cs-CZ
ms.lasthandoff: 04/23/2019
ms.locfileid: "61687328"
---
# <a name="compiler-error-cs0156"></a>Compiler Error CS0156
Příkaz throw bez argumentů není povolený v nakonec klauzuli, která je vnořená do nejbližší uzavírající klauzule catch  
  
 A [throw](../../csharp/language-reference/keywords/throw.md) příkaz bez parametrů může být použit pouze v **catch** klauzuli, která nepřijímá žádné parametry.  
  
 Další informace najdete v tématu [příkazy zpracování výjimek](../../csharp/language-reference/keywords/exception-handling-statements.md) a [výjimek a zpracování výjimek](../../csharp/programming-guide/exceptions/index.md).  
  
 Následující ukázka generuje CS0156:  
  
```csharp  
// CS0156.cs  
using System;  
  
namespace MyNamespace  
{  
   public class MyClass2 : Exception  
   {  
   }  
  
   public class MyClass  
   {  
      public static void Main()  
      {  
         try  
         {  
            throw;   // CS0156  
         }  
  
         catch(MyClass2)  
         {  
            throw;   // this throw is valid  
         }  
      }  
   }  
}  
```
